editor/js/Sidebar.Settings.Shortcuts.js

```diff
@@ -41,6 +42,23 @@ function SidebarSettingsShortcuts( editor ) {
 
 				config.setKey( configName, value );
 
+				const otherShortcuts = shortcuts.filter( shortcut => shortcut !== name );
+				for ( let i = 0; i < otherShortcuts.length; i ++ ) {
+
+					const shortcut = otherShortcuts[ i ];
+					const configName = 'settings/shortcuts/' + shortcut;
+					const hotkey = config.getKey( configName );
+
+					if ( hotkey === value ) {
+
+						config.setKey( configName, '' );
+
+					}
+
+				}
+
+				refreshUI();
+
```

Comment:
If the developer assigns multiple keys to different actions, the app does not crash or throw a runtime error. The first action assigned to a key is executed, others are ignored. That is acceptable, imo.

TBH, the additional code does not really justify the improvement in usability.